schema/rdl_schema_0.1.json

```
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
```

This is a difficult description to author because CKAN is pretty loose about what a resource is. However, I don't think 'version' is quite the right term. I've tried to crib something based on the discussion and resources linked in #54

```suggestion
			"description": "An individual data file, or other form of data, that is part of a dataset. Resources can be fully informationally equivalent, such as CSV and JSON serializations of the same data. Resources can also have different levels of fidelity, such as a CSV file and a graphical representation of the data within the file. Resources can also differ in other aspects, such as the temporal coverage of each representation.",
```